contrail-overlay.nix

```diff
@@ -253,9 +256,13 @@ in rec {
     };
   });
 
-  contrail50 = contrail41.overrideScope' (lself: lsuper: {
+  contrail50 = contrail41.overrideScope' (lself: lsuper: rec {
     contrailVersion = "5.0";
     contrailSources = callPackage ./sources-R5.0.nix { };
+    webui = callPackages ./pkgs/webui.nix { sources = contrailSources; };
```

Comment:
If you use `lsuper.callPackage` then you don't need to pass `sources` and in `pkgs/webui.nix` you can get `contrailSources`. Same as `pkgs/workspace.nix` for example.